x-pack/test/api_integration/deployment_agnostic/apis/observability/dataset_quality/integrations.ts

```diff
@@ -57,6 +58,12 @@ export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
           withInternalHeaders: true,
         }
       );
+      const body = await callApiAs({
+        roleScopedSupertestWithCookieCredentials: supertestAdminWithCookieCredentials,
+      });
+      preExistingIntegrations = body.integrations.map(
+        (integration: Integration) => integration.name
+      );
```

Comment:
Optionally I would move into 1 line since it is a pre-condition and it is fair to cut the hook code

```
preExistingIntegrations = (await callApiAs({ roleScopedSupertestWithCookieCredentials: supertestAdminWithCookieCredentials })).integrations.map((integration: Integration) => integration.name);
```